tsconfig.json

```diff
@@ -32,6 +32,7 @@
       "@pipes/*": ["src/app/pipes/*"],
       "@utils/*": ["src/app/utils/*"],
       "@components/*": ["src/app/components/*"],
+      "@widgets/*": ["src/app/widgets/*"],
```

Comment:
Minor: I think it's so strange to use it in DI framework.

Reply:
This is a common practice for using alias in paths in `tsconfig.ts` file.

FYI. Check It - https://stackoverflow.com/questions/43281741/how-to-use-paths-in-tsconfig-json.